resources/zib-Patient.xml

```diff
@@ -0,0 +1,468 @@
+  <purpose value="This Patient resource represents the Dutch zib ('Zorginformatiebouwsteen', i.e. Health and Care Information Model) [Patient v3.2 (2020)](https://zibs.nl/wiki/Patient-v3.2(2020EN)), [Nationality v3.0 (2020)](https://zibs.nl/wiki/Nationality-v3.0(2020EN)) and [MaritalStatus v3.1 (2020)](https://zibs.nl/wiki/MaritalStatus-v3.1(2020EN))." />
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="rim" />
+    <uri value="http://hl7.org/v3" />
+    <name value="RIM Mapping" />
+  </mapping>
+  <mapping>
+    <identity value="cda" />
+    <uri value="http://hl7.org/v3/cda" />
+    <name value="CDA (R2)" />
+  </mapping>
+  <mapping>
+    <identity value="w5" />
+    <uri value="http://hl7.org/fhir/fivews" />
+    <name value="FiveWs Pattern Mapping" />
+  </mapping>
+  <mapping>
+    <identity value="v2" />
+    <uri value="http://hl7.org/v2" />
+    <name value="HL7 v2 Mapping" />
+  </mapping>
+  <mapping>
+    <identity value="loinc" />
+    <uri value="http://loinc.org" />
+    <name value="LOINC code for the element" />
+  </mapping>
+  <mapping>
+    <identity value="zib-patient-v3.2-2020EN" />
+    <uri value="https://zibs.nl/wiki/Patient-v3.2(2020EN)" />
+    <name value="zib Patient-v3.2(2020EN)" />
+  </mapping>
+  <mapping>
+    <identity value="zib-nationality-v3.0-2020EN" />
+    <uri value="https://zibs.nl/wiki/Nationality-v3.0(2020EN)" />
+    <name value="zib Nationality-v3.0(2020EN)" />
+  </mapping>
+  <mapping>
+    <identity value="zib-maritalstatus-v3.1-2020EN" />
+    <uri value="https://zibs.nl/wiki/MaritalStatus-v3.1(2020EN)" />
+    <name value="zib MaritalStatus-v3.1(2020EN)" />
+  </mapping>
+  <mapping>
+    <identity value="zib-languageproficiency-v3.2-2020EN" />
+    <uri value="https://zibs.nl/wiki/LanguageProficiency-v3.2(2020EN)" />
+    <name value="zib LanguageProficiency-v3.2(2020EN)" />
+  </mapping>
+  <kind value="resource" />
+  <abstract value="false" />
+  <type value="Patient" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/Patient" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="Patient">
+      <path value="Patient" />
+      <short value="Patient" />
+      <definition value="Root concept of the Patient information model. This root concept contains all data elements of the Patient information model." />
```

Comment:
Mm! Ik ben niet zo fan van deze kopie van definition. Kan daar de concept naam van de Patient?

Reply:
Conceptnaam?

Dit is volgens de guidelines wat er in definition hoort te staan. Het stukje tekst onder 'Concept' op de zib pagina staat in StructureDefinition.description

Of we moeten een uitzondering voor rootconcepten maken (want die definition is meestal dit en weinigzeggend)

Reply:
Ja! Dit hadden we volgens mij besproken. Ik heb het ook doorgevoerd in de zib-compliance tool, maar gek genoeg niet opgeschreven in de profiling guidelines. Mea Culpa.

Reply:
Top!